src/chrome/content/rules/Pixlr.com.xml

```diff
@@ -0,0 +1,19 @@
+<ruleset name="Pixlr.com" platform="mixedcontent">
+
+	<target host="pixlr.com" />
+	<target host="www.pixlr.com" />
+	<target host="api.pixlr.com" />
+	<target host="cdn.pixlr.com" />
+	<target host="img-cdn.pixlr.com" />
+		<test url="http://img-cdn.pixlr.com/images/pixlr-favicon-16x16.ico" />
+	<target host="support.pixlr.com" />
+	
+	<!-- Mixed content examples -->
+		<test url="http://pixlr.com/blog/support/" />
```

Comment:
You can add an exclusion for `^http://pixlr\.com/blog/` and remove the platform.